.github/workflows/main.yml

```diff
@@ -16,11 +16,13 @@ on:
   merge_group:
     types: [checks_requested]
 
+env:
+  NX_CLOUD_ACCESS_TOKEN: ${{ github.event_name == 'push' && github.ref != 'main' && '' || secrets.NX_CLOUD_ACCESS_TOKEN }}
```

Comment:
this will always return `secrets.NX_CLOUD_ACCESS_TOKEN`
because `''` is falsy

Reply:
indeed